llvm/lib/Transforms/Instrumentation/GCOVProfiling.cpp

```diff
@@ -166,6 +170,7 @@ class GCOVProfiler {
   std::vector<Regex> ExcludeRe;
   DenseSet<const BasicBlock *> ExecBlocks;
   StringMap<bool> InstrumentedFiles;
+  IntrusiveRefCntPtr<vfs::FileSystem> VFS;
```

Comment:
I think `GCOVProfiler` lives inside the pass so this doesn't need to be ref-counted?